zappa/handler.py

```diff
@@ -383,8 +383,7 @@ def handler(self, event, context):
             whole_function = event["command"]
             app_function = self.import_module_and_get_function(whole_function)
             result = self.run_function(app_function, event, context)
-            print("Result of %s:" % whole_function)
-            print(result)
+            logger.debug("Result of %s: %s", whole_function, result)
```

Comment:
No need to combine the result and the message, sometimes we want this output separate.

Reply:
If you image a cloud environment with lots of lambdas running in parallel, it can be difficult to connect isolated `logger.debug(result)` log prints to the correct context. What would be the reason for wanting the result in a separate log message?